data/mods/MindOverMatter/effectoncondition/eoc_mental_engineering.json

```diff
@@ -0,0 +1,5106 @@
+          "As much as it hurts you you just cannot get over the dullness of familiar places.  With a bit of psychic surgery, that dullness can get a good bit brighter.",
+          "Even with your already incredible desire to keep moving you feel this may be a bad idea.  Ensure that you will never stop moving.",
+          "Even taking the time to consider this alteration of your head is making you incredibly unhappy at your lack of travel.  You should really fix this.",
+          "Wearing tainted clothing does seem pretty gross.  Definitely really bad for your health too.  Make sure you don't have to worry about that by making sure that you're mentally incapable of ever wearing them.",
+          "Logically speaking, your refusal to wear tainted clothing really isn't that bad.  There may be some benefit to reducing your squeamishness, though.",
+          "Finding time to DM a game is probably not going to happen again.  Even so, by spending an inordinate amount of time studying a DM'ing guide while under the effect of your psychic powers you think you can get some ideas on how to do it again but quicker.",
+          "RPG board gaming is for nerds!  Make yourself not a nerd by giving yourself a tiny bit of brain damage to emphasize your desire to do other things and reduce your ability to be a nerd!",
+          "The government is gone and so are the owners of all these buildings.  And trees.  And really everything flamable.  You think you can make yourself enjoy the act a lot more than you may already do, but this may have some side effects.",
+          "As fun as it is, you can begrugingly agree that there's better things to do than setting fires.  Put a hypothetical therapist out of business with your psychic powers and fix that.",
+          "Maybe people will trust you more if you're visibly terrible at telling the slightest lie?  Who knows.",
+          "Being able to lie could be an important part of your life... if there was even anyone to lie to.  You doubt this would help you much.",
+          "There's still a couple of survivors around, and being good at lying to them might give you some opportunities you otherwise would lack.  It might also get you in trouble",
+          "Reduce your ability to lie with some damaging psychic surgery.  May or may not make you a better person.",
+          "Turn what would normally be a personal decision into a psychically mandated behavior.  You might starve to death, but at least you won't hurt any animals while doing so.",
+          "Even the thought of doing this makes you incredibly uncomfortable, but of all the reasons to quite veganism, the complete destruction of your entire world probably qualifies.",
+          "Logically speaking, those things that kind of look human really aren't.  It might currently feel uncomfortable, but its hard to think of a real argument against eating them.  May as well get over your emotions.  Telepathically!",
```

Comment:
```suggestion
          "Logically speaking, those things that kind of look human really aren't.  It might currently feel uncomfortable, but it's hard to think of a real argument against eating them.  May as well get over your emotions.  Telepathically!",
```